template-files/android/AndroidManifest.xml

```diff
@@ -367,6 +367,8 @@
     <!-- The Facebook SDK runs FacebookInitProvider on startup and crashes if there isn't an ID here -->
     <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="fb0"/>
 
+    <meta-data android:name="host.exp.exponent.InExpoClient" android:value="true"/>
```

Comment:
This file defines both Expo Client manifest and apps built in Turtle, so this value would always be `true`